doc/nrf/config_and_build/configuring_app/zephyr_samples_sysbuild.rst

```diff
@@ -13,7 +13,7 @@ For applications and samples in the |NCS| repository, this is handled automatica
 +---------------------------------------------------------+-------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
 | :kconfig:option:`SB_CONFIG_NETCORE_HCI_IPC`             | ``hci_ipc``             | ✓         | ✕        |                                                                                                                       |
 +---------------------------------------------------------+-------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
-| :kconfig:option:`SB_CONFIG_NETCORE_RPC_HOST`            | ``rpc_host``            | ✓         | ✕        | Requires that application be setup for this mode                                                                      |
+| :kconfig:option:`SB_CONFIG_NETCORE_RPC_HOST`            | ``rpc_host``            | ✓         | ✕        | Requires that application is setup for this mode                                                                      |
```

Comment:
This is gramatically and linguistically correct :) https://en.wikipedia.org/wiki/English_subjunctive

Reply:
Ok, removing :)